appdatabase/migrations/sql/1685134628_transfers_receipt_json_blob_out.down.sql

```diff
@@ -0,0 +1 @@
+-- This migration is in GO code. If GO migration failure this serves to rollback to the previous one
```

Comment:
Just a note about our migrations, we don't ever roll a migration back. This is because we can't control the clients and so have no control over what version the users have.

Reply:
As mentioned in the previous answer, this is required only in the case of custom migrations as an alternative to the API limitations of the migration system to unroll the changes made by the custom code in case of failures. Otherwise, the DB migration will mark the current custom version as run after running the SQL "marker" migration. Hence after fixing the custom migration, it won't be rerun, and to fix will have to create another migration. Long story short, it is a feature to make fixing Golang migrations easy and optional.

Reply:
With this approach, we have a relatively simple and robust way of running Golang migrations along when there is no way around it.

Reply:
Ah ok, @stefandunca I see. Rollback in the event the go code fails, that makes sense to me. Thank you for your patience in explaining this to me.